hack/e2e.go

```diff
@@ -39,6 +39,7 @@ var (
 	version          = flag.String("version", "", "The version to be tested (including the leading 'v'). An empty string defaults to the local build, but it can be set to any release (e.g. v0.4.4, v0.6.0).")
 	up               = flag.Bool("up", false, "If true, start the the e2e cluster. If cluster is already up, recreate it.")
 	push             = flag.Bool("push", false, "If true, push to e2e cluster. Has no effect if -up is true.")
+	checkpush        = flag.Bool("checkpush", false, "If true, push to e2e cluster asserting that the cluster is stable during upgrade.")
```

Comment:
I assume there's an obvious reason and this is probably a dumb question, but do we *need* another top-level flag for this test? It seems like because the test is already skipped by default we could just specify that we want to target it on the command line explicitly when we want to have it run?